src/glossary.md

```diff
@@ -0,0 +1,39 @@
+# Glossary
+
+## Toronto Community Network
```

Comment:
Should we also include acronyms like "tcn"

Reply:
That acronym is not formalized, so I left it for now.

Reply:
Perhaps it needs to be,

It is being used in FQDN naming
https://github.com/tomeshnet/toronto-community-network/issues/21

> For example a FQDN (fully qualified domain name) will be sn1a1.core.tcn.tomesh.net for a device operated by the core team at Toronto Community Network. The node will also answer as sn1a1.tcn.tomesh.net

Reply:
Yes. I was sort of waiting for someone else to formalize it, but I'll add it here.